packages/frontend/src/pages/instruct-lab/AboutInstructLabExploreCard.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
```

Comment:
Don't put the component in the `page/` folder, this is only for pages, add an `instructlab` section in the lib folder